cirq-web/cirq_web/widget_test.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2021 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+import cirq_web
+
+
+class MockWidget(cirq_web.Widget):
```

Comment:
```suggestion
class TestWidget(cirq_web.Widget):
```

Mock evokes different concepts - like capturing call arguments, using them as an intermediary...I'd just call it `TestWidget`

Reply:
TestWidget gives a pytest warning..got stuck wondering why for like 30 mins lol, but its because if the name starts with Test pytest doesn't skip it